tests/ci/cherry_pick.py

```diff
@@ -67,15 +67,17 @@ class ReleaseBranch:
 ### Note
 
 This pull-request will be merged automatically as it reaches the mergeable state, \
-**do not merge it manually**.
+**do not merge it manually**. It's 100% safe, but completely meaningless.
```

Comment:
I don't know the context of this PR but to outside readers it is not clear *what* is "100% safe but completely meaningless." Do you mean
- the cherry-pick PR?
- the automatic merge?
- the manual merge?

I guess it is the latter. Then please say

```
This pull-request will be merged automatically.
Please do not merge it manually (but if you accidentally did, nothing bad will happen).
```

Reply:
L. 72: "If the PR was manually reopened after being closed."

(besides that, it is unclear how l. 74 - 80 (PR stuck or screwed) are connected to l. 72 (PR reopened))